client/lcd/test_helpers.go

```diff
@@ -190,6 +191,7 @@ func InitializeTestLCD(t *testing.T, nValidators int, initAddrs []sdk.AccAddress
 	node, err := startTM(config, logger, genDoc, privVal, app)
 	require.NoError(t, err)
 
+	time.Sleep(3 * time.Second)
```

Comment:
Hmm, what's with this? Can we instead wait for Tendermint to return a response to a query or something?